package.json

```diff
@@ -29,30 +29,30 @@
         "url": "https://github.com/equinor/EchoCore"
     },
     "devDependencies": {
-        "@babel/core": "^7.20.2",
+        "@babel/core": "^7.20.12",
         "@babel/preset-env": "^7.20.2",
         "@babel/preset-react": "^7.18.6",
         "@babel/preset-typescript": "^7.18.6",
         "@equinor/echo-update": "^1.3.1",
-        "@types/jest": "^28.1.8",
-        "@types/node": "^18.11.9",
-        "@types/react": "^17.0.52",
-        "@typescript-eslint/eslint-plugin": "^5.44.0",
-        "@typescript-eslint/parser": "^5.44.0",
-        "babel-jest": "^27.5.1",
+        "@types/jest": "^29.2.6",
+        "@types/node": "^18.11.18",
+        "@types/react": "^17.0.53",
+        "@typescript-eslint/eslint-plugin": "^5.48.2",
+        "@typescript-eslint/parser": "^5.48.2",
+        "babel-jest": "^29.3.1",
         "babel-plugin-polyfill-corejs3": "^0.6.0",
         "babel-plugin-polyfill-regenerator": "^0.4.1",
-        "eslint": "^8.28.0",
-        "eslint-config-prettier": "^8.5.0",
+        "eslint": "^8.32.0",
+        "eslint-config-prettier": "^8.6.0",
         "eslint-plugin-prettier": "^4.2.1",
-        "eslint-plugin-react": "^7.31.11",
+        "eslint-plugin-react": "^7.32.1",
         "eslint-plugin-react-hooks": "^4.6.0",
-        "jest": "^27.5.1",
-        "jest-expo": "^44.0.1",
-        "lerna": "^4.0.0",
-        "prettier": "^2.7.1",
+        "jest": "^29.3.1",
```

Comment:
Seems like the tests fail because you need jest-environment-jsdom or something like that when upgrading past jest 28.

Reply:
Yes, fixed by adding it, and also mocking globally @microsoft/applicationinsights-web